pkg/ottl/ottlconfig/config.go

```diff
@@ -14,14 +14,14 @@
 
 package ottlconfig // import "github.com/open-telemetry/opentelemetry-collector-contrib/pkg/ottl/ottlconfig"
 
-// Config configures the OTTL queries to execute against traces, metrics, and logs.
+// Config configures the OTTL statements to execute against traces, metrics, and logs.
 type Config struct {
 	Traces  SignalConfig `mapstructure:"traces"`
 	Metrics SignalConfig `mapstructure:"metrics"`
 	Logs    SignalConfig `mapstructure:"logs"`
 }
 
-// SignalConfig configures OTTL queries to execute.
+// SignalConfig configures OTTL statements to execute.
 type SignalConfig struct {
-	Queries []string `mapstructure:"queries"`
+	Statements []string `mapstructure:"statements"`
 }
```

Comment:
@TylerHelmuth @evan-bradley  I am not sure if we should have these 2 structs here at all. My view is that we will also have a "Condition" version of this for components that only need the condition. Also there will likely be components that only work on some of the data signals not on all. Because of that I believe this is not that useful.

I would suggest to try to enforce this structure, but not by having this message, and only in reviews. Happy to hear what others think.

Reply:
Yes I anticipate the structure is going to change significantly with the introduction of new contexts (or how we utilize existing contexts (#14578)) and for Conditions (#13545)

Reply:
@TylerHelmuth How do you see these structs being used in components after the changes you mentioned? I agree with @bogdandrutu, to me the config structure for a component using the OTTL will often be specific enough that these may not provide much value.

Reply:
It is likely these structs will go away.

Reply:
Lets remove it from here then.

Reply:
I've removed it. It seemed prudent to also update the `metrics` key to `datapoints` since we're already making a breaking change in the config structure. Let me know if we want to keep it as `metrics` for now with https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/14578 on the horizon.